src/cpp/core/http/Response.cpp

```diff
@@ -107,6 +107,23 @@ void Response::setNoCacheHeaders()
              "no-cache, no-store, max-age=0, must-revalidate");
 }
 
+void Response::setFrameOptionHeaders(frame_options::XFrameOptions options)
+{
+   std::string option;
+   if (options == frame_options::Deny)
```

Comment:
FWIW since this an `enum`, you could use a `switch` statement and (modern) compilers will warn if not all possible `enum` values are handled. (Ie, this could be a compile-time instead of run-time warning)